packages/view-types/statistical-plots/src/DotPlotSubscriber.js

```diff
@@ -0,0 +1,241 @@
+import React, { useMemo } from 'react';
+import {
+  TitleInfo,
+  useCoordination, useLoaders,
+  useUrls, useReady, useGridItemSize,
+  useFeatureSelection, useObsSetsData,
+  useObsFeatureMatrixIndices,
+  useFeatureLabelsData,
+  registerPluginViewType,
+} from '@vitessce/vit-s';
+import { ViewType, COMPONENT_COORDINATION_TYPES } from '@vitessce/constants-internal';
+import { VALUE_TRANSFORM_OPTIONS, getValueTransformFunction } from '@vitessce/utils';
+import { treeToObsIndicesBySetNames, mergeObsSets } from '@vitessce/sets-utils';
+import { mean } from 'd3-array';
+import uuidv4 from 'uuid/v4';
+import CellSetExpressionPlotOptions from './CellSetExpressionPlotOptions';
+import DotPlot from './DotPlot';
+import { useStyles } from './styles';
+
+/**
+ * Get expression data for the cells
+ * in the selected cell sets.
+ * @param {object} expressionMatrix
```

Comment:
For the given gene selection, and also it doesn't have the rows and cols I think.